sdk/clientcore/annotation-processor-test/src/main/java/io/clientcore/annotation/processor/test/implementation/TestInterfaceClientImpl.java

```diff
@@ -93,7 +93,7 @@ Response<FooListResult> listNextFooListResult(@PathParam(value = "nextLink", enc
                                                       RequestOptions requestOptions);
 
         @HttpRequestInformation(method = HttpMethod.GET, path = "foos", expectedStatusCodes = { 200 })
-        Response<List<Foo>> listFoo(@HostParam("uri") String uri, RequestOptions requestOptions);
+        Response<List<Foo>> listFoo(@HostParam("uri") String uri, @QueryParam(value = "tags", multipleQueryParams = true) List<String> tags, RequestOptions requestOptions);
```

Comment:
Not related to this PR, but is there a PR coming up for when `@QueryParam` has `encoded = true`?

Reply:
yep, next PR. It should add support for encoded path and query params.